A.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: yellowgreen;
+}
```

Comment:
- [OPTIONAL] Great job here 👌💯 but according to the requirements, you should use a class to select the `<h1>` tag, So Kindly add a class name to this element `<h1 class="header">` in your HTML and use it in your CSS file 
`.header {color: yellowgreen;}`👌


![image](https://user-images.githubusercontent.com/45231276/185032956-ca65df88-2c93-4fdc-97c0-93e7050d2989.png)